terraform-plan.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash ./bids/shell.nix
+set -eux
+
+scriptroot=$(dirname "$(realpath "$0")")
+cd "$scriptroot/bids"
+
+set +e
+terraform init
+terraform plan -detailed-exitcode -input=false -out ./terraform.plan
+exitcode=$?
+set -e
+
+if [ "$exitcode" -eq 2 ]; then
+  echo "Diff present, uploading pipline apply stage."
+
+  (
+    echo "steps:"
```

Comment:
Let's move this in to the `cat` block. In the ofborg repo it makes sense since we're doing code gen in a loop. Here, we can more easily prioritize readability.

Reply:
Actually, let's make this its `.yml` file that gets uploaded.